pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTableRestletResource.java

```diff
@@ -636,7 +636,7 @@ public RebalanceResult rebalance(
           + "segments in a round-robin fashion as if adding new segments to an empty table)") @DefaultValue("false")
   @QueryParam("bootstrap") boolean bootstrap,
       @ApiParam(value = "Whether to allow downtime for the rebalance") @DefaultValue("false") @QueryParam("downtime")
-          boolean downtime, @ApiParam(
+      boolean downtime, @ApiParam(
```

Comment:
(nit) The format doesn't seem correct for this method

Reply:
Does not look too odd, but reformatted.